### ISSUE TITLE
unclear deprecation warnings

### ISSUE THREAD

**Issue by angelo-v** [philpep/testinfra#204]
Since upgrading to latest testinfra all my test spit out the following warning
> get_module() is deprecated, use new host API

I cannot find information in the documentation about what exactly I am expected to do.

I am already using the host fixture. My tests use docker connection backend and configure hosts on a per module basis via testinfra_hosts= ["docker://container-name"]. Same problem with some tests using ansible backend.

**Comment by philpep** [comment]
Hello, this warning is raised when you have a instance of TestinfraBackend and you get a module either by `backend.module` or `backend.get_module(module)`. I cannot say more without looking at your test code.

**Comment by wicksy** [comment]
I get similar running a simple package test based on one of the examples:

```
[vagrant@ocptest ~]$ cat test.py
import pytest

@pytest.mark.parametrize("name,version", [
    ("python", "2.7"),
])
def test_packages(host, name, version):
    pkg = host.package(name)
    assert pkg.is_installed
    assert pkg.version.startswith(version)
[vagrant@ocptest ~]$ testinfra -v ./test.py
============================================================================================================ test session starts ============================================================================================================
platform linux2 -- Python 2.7.5, pytest-3.1.0, py-1.4.33, pluggy-0.4.0 -- /bin/python
cachedir: .cache
rootdir: /home/vagrant, inifile:
plugins: testinfra-1.6.2
collected 1 items

test.py::test_packages[local-python-2.7] PASSED

============================================================================================================= warnings summary ==============================================================================================================
None
  Module already imported so can not be re-written: testinfra

test.py::test_packages[local-python-2.7]
  /usr/lib/python2.7/site-packages/testinfra/modules/package.py:66: DeprecationWarning: get_module() is deprecated, use new host API
    if host.system_info.type == "freebsd":
  /usr/lib/python2.7/site-packages/testinfra/modules/base.py:26: DeprecationWarning: get_module() is deprecated, use new host API
    "run_expect": _host.run_expect,

-- Docs: http://doc.pytest.org/en/latest/warnings.html
=================================================================================================== 1 passed, 3 warnings in 0.12 seconds ====================================================================================================
[vagrant@ocptest ~]$
```

**Comment by angelo-v** [comment]
For examples of tests that produce this warning please take a look at
https://github.com/angelo-v/docker-testinfra/tree/master/test

The output is:

```
=========================================== warnings summary ============================================
test/test_image.py::test_pip_packages[docker://test_container]
  /usr/lib/python2.7/site-packages/testinfra/modules/base.py:26: DeprecationWarning: get_module() is deprecated, use new host API
    "run_expect": _host.run_expect,

-- Docs: http://doc.pytest.org/en/latest/warnings.html
================================= 2 passed, 1 warnings in 2.67 seconds ==================================
```

**Comment by philpep** [comment]
Thanks for code examples. The issue is visible since pytest 3.1 release because it now show warnings.

This been fixed and released in testinfra 1.6.3
